src/Components/App/App.js

```diff
@@ -69,6 +92,23 @@ class App extends Component {
       })
       .catch(err => err.message)
   }
+
+  handleChange = (event) => {
+    event.preventDefault();
+    let searchValue = (event.target.value).toUpperCase();
+    let matches = this.filterMovies(searchValue);
+    this.setState({ matchingMovies: matches });
+    console.table(matches)
+
+  }
+
+  filterMovies = (searchValue) => {
+    return this.state.movies.filter(movie => {
+      let capTitle = movie.title.toUpperCase();
```

Comment:
Here we need to make sure that both the initial letters typed go to lower case and the the search word in complete form are being set to lower case.